aws-cli-v1.yaml

```diff
@@ -1,6 +1,6 @@
 package:
-  name: aws-cli
```

Comment:
I think we want to have this `provides: ['aws-cli=1.27.999']` and for v2 to `provides: ['aws-cli=2.X.999']` so that folks that `apk add aws-cli` still get the relevant packages.

Otherwise this LGTM!

Is v1 still receiving updates, specifically security updates? 🤔

Reply:
aws cli v1 is GA still https://docs.aws.amazon.com/sdkref/latest/guide/version-support-matrix.html